addons/mail/models/res_partner.py

```diff
@@ -24,6 +25,8 @@ class Partner(models.Model):
     channel_ids = fields.Many2many('mail.channel', 'mail_channel_partner', 'partner_id', 'channel_id', string='Channels', copy=False)
     # override the field to track the visibility of user
     user_id = fields.Many2one(tracking=True)
+    peer_token = fields.Char(string='Peer Token')
+    room_id = fields.Many2one('mail.room')
```

Comment:
Adding fields in partner is always very sensitive due to the amount of records there is. Especially if those fields won't be used for 99% of them.
To double check why we need them, and if we can't put them somewhere else?

Also how will this work with guests?